x-pack/plugin/graph/src/main/java/org/elasticsearch/xpack/graph/rest/action/RestGraphAction.java

```diff
@@ -64,15 +62,10 @@
     public RestGraphAction(Settings settings, RestController controller) {
         super(settings);
 
-        // @deprecated Remove in 7.0
-        controller.registerWithDeprecatedHandler(GET, "/{index}" + URI_BASE + "/graph/_explore", this,
-                                                 GET, "/{index}" + URI_BASE + "/_graph/_explore", deprecationLogger);
-        controller.registerWithDeprecatedHandler(POST, "/{index}" + URI_BASE + "/graph/_explore", this,
-                                                 POST, "/{index}" + URI_BASE + "/_graph/_explore", deprecationLogger);
-        controller.registerWithDeprecatedHandler(GET, "/{index}/{type}" + URI_BASE + "/graph/_explore", this,
-                                                 GET, "/{index}/{type}" + URI_BASE + "/_graph/_explore", deprecationLogger);
-        controller.registerWithDeprecatedHandler(POST, "/{index}/{type}" + URI_BASE + "/graph/_explore", this,
-                                                 POST, "/{index}/{type}" + URI_BASE + "/_graph/_explore", deprecationLogger);
+        controller.registerHandler(GET, "/{index}" + URI_BASE + "/graph/_explore", this);
+        controller.registerHandler(POST, "/{index}" + URI_BASE + "/graph/_explore", this);
+        controller.registerHandler(GET, "/{index}/{type}" + URI_BASE + "/graph/_explore", this);
+        controller.registerHandler(POST, "/{index}/{type}" + URI_BASE + "/graph/_explore", this);
```

Comment:
Technically speaking the doc `type` in urls is being deprecated in 7.0. The general pattern we're adopting is it's not flagged as deprecated if you also pass an `include_type_name=false` along with the request.
I can pick this up as part of types removal work if you want to leave that out for now.

Reply:
@markharwood Yeah, I would rather leave the typeless work separate from this change; thanks for offering to pick that up. My aim here is only to remove the already deprecated endpoints. I am going to turn around in a follow-up and add new endpoints (`/_graph/explore`) and deprecate all endpoints that have `/{index}/_xpack`.